app/repositories/__init__.py

```
@@ -0,0 +1,16 @@
+from .user import UserRepository
+from .post import PostRepository
+from .comment import CommentRepository
+from .like import LikeRepository
+
+__all__ = [
+    "BaseRepository",
+    "UserRepository",
+    "PostRepository",
+    "CommentRepository",
+    "LikeRepository"
+]
```

Ruff (0.14.11)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)